src/event-log/event-log.ts

```diff
@@ -0,0 +1,32 @@
+export type Event = {
+  watermark: string,
+  messageCid: string
+};
+
+export interface EventLog {
+ /**
+  * opens a connection to the underlying store
+  */
+  open(): Promise<void>;
+
+  /**
+   * closes the connection to the underlying store
+   */
+  close(): Promise<void>;
+
+  /**
+   * adds an event to a tenant's event log
+   * @param tenant - the tenant's DID
+   * @param messageCid - the CID of the message
+   */
+  append(tenant: string, messageCid: string): Promise<string>
+
+  /**
+   * retrieves all of a tenant's events that occurred after the watermark provided.
+   * If no watermark is provided, all events for a given tenant will be returned.
+   *
+   * @param tenant
+   * @param watermark
+   */
+  getEventsAfter(tenant: string, watermark?: string): Promise<Array<Event>>
```

Comment:
NIT: what about just `get`?  the `After` part is a bit misleading IMO in the case where someone just calls `getEventsAfter(tenant)` since there is no `watermark` argument

Reply:
> the `After` part is a bit misleading IMO in the case where someone just calls

fair. though i think `get` is misleading as well because it i'm not sure whether i'm getting 1 or many. i suppose one could make the argument that the return signature would provide you with that detail.

Reply:
@dcrousso, refactored method signature. [diff](https://github.com/TBD54566975/dwn-sdk-js/commit/902d7babcfcc2cfa3cc34d1bc944e7e3448741a1#diff-1107f46751ce21d33b21d75810446ddf320ec0bc8b7170b3f8ccbefa4194c2e3L54-R59). lmk what you think

Reply:
allowing the client to provide any `gt` they want might be a bit vague/permissive, but since this is really only meant to be used internally it's probably fine :)

also it's not like they could do much else with the `gt` since all of the keys (after the `partition` based on the `tenant`) would be a `watermark` anyways